string_grouper/test/test_string_grouper.py

```diff
@@ -5,7 +5,38 @@
 from string_grouper.string_grouper import DEFAULT_MIN_SIMILARITY, \
     DEFAULT_MAX_N_MATCHES, DEFAULT_REGEX, \
     DEFAULT_NGRAM_SIZE, DEFAULT_N_PROCESSES, DEFAULT_IGNORE_CASE, \
-    StringGrouperConfig, StringGrouper, StringGrouperNotFitException
+    StringGrouperConfig, StringGrouper, StringGrouperNotFitException, \
+    match_most_similar, group_similar_strings, match_strings
+
+
+class SimpleExample(object):
+    def __init__(self):
+        self.customers_df = pd.DataFrame(
+           [
+              ('BB016741P', 'Mega Enterprises Corporation', 'Address0', 'Tel0', 'Description0', 0.2),
+              ('CC082744L', 'Hyper Startup Incorporated', '', 'Tel1', '', 0.5),
+              ('AA098762D', 'Hyper Startup Inc.', 'Address2', 'Tel2', 'Description2', 0.3),
+              ('BB099931J', 'Hyper-Startup Inc.', 'Address3', 'Tel3', 'Description3', 0.1),
+              ('HH072982K', 'Hyper Hyper Inc.', 'Address4', '', 'Description4', 0.9),
+              ('EE059082Q', 'Mega Enterprises Corp.', 'Address5', 'Tel5', 'Description5', 1.0)
+           ],
+           columns=('Customer ID', 'Customer Name', 'Address', 'Tel', 'Description', 'weight')
+        )
+        # group_similar_strings(
```

Comment:
Comments can be removed?

Reply:
They can.  Though I thought they provided context for the expected result displayed on the next line.